runtime/src/main/java/org/corfudb/runtime/view/StreamOptions.java

```diff
@@ -0,0 +1,16 @@
+package org.corfudb.runtime.view;
+
+/**
+ * Created by maithem on 6/20/17.
+ */
+public class StreamOptions {
```

Comment:
From a usability perspective, `new StreamOptions(true)` is not very intuitive, especially with multiple options. Either the fluent pattern or setters would be much nicer.